models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
+| Settings                    |                                                             |
++=============================+=============================================================+
+| ``DATA_ASSIMILATION_LND``   | If ``.TRUE.`` data assimilation is enabled.  If ``.FALSE.`` |
+|                             | no data assimilation is performed, and simulation is        |
+|                             | performed as a normal CLM run.                              |
++-----------------------------+-------------------------------------------------------------+
+| ``DATA_ASSIMILATION_SCRIPT``| Location of script ``assimilate.csh`` that performs the     |
+|                             | assimilation.  Controls execution of DART scripts and passes|
+|                             | files between DART and CLM code.                            |
++-----------------------------+-------------------------------------------------------------+
+| ``STOP_OPTION``             | Unit of time that controls duration of each assimilation    |
+|                             | cycle (see ``STOP_N``)                                      |
++-----------------------------+-------------------------------------------------------------+
+| ``STOP_N``                  | Duration of each assimilation cycle in units of time        |
+|                             | defined by ``STOP_OPTION``                                  |
++-----------------------------+-------------------------------------------------------------+
+| ``DATA_ASSIMILATION_CYCLES``| The number of assimilation cycles performed for a single    |
+|                             | job submission                                              |
++-----------------------------+-------------------------------------------------------------+
+| ``RESUBMIT``                | The number of times each job submission is repeated as      |
+|                             | defined by ``STOP_OPTION``, ``STOP_N``, and                 |
+|                             | ``DATA_ASSIMILATION_CYCLES``                                |
++-----------------------------+-------------------------------------------------------------+
+| ``CONTINUE_RUN``            | If ``.FALSE.`` this is the startup of the assimilation run  |
+|                             | beginning from the ``refcase``.  If ``.TRUE.`` this is a    |
+|                             | continuation of a previous assimilation run.                |
++-----------------------------+-------------------------------------------------------------+ 
+
+ 
+Before submission review your ``input.nml`` within your case folder to confirm
+the settings reflect those from previous steps of the tutorial.  Next submit
+the assimilation run:
+
+::
+
+ > cd <caseroot>
+ > ./case.submit
+
+Check the status of the job on Cheyenne using PBS commands to determine if job
+is queued (Q), running (R) or completed.
+
```

Comment:
I'm at this stage at the moment. My filter is failing at the `fill_inflation_restart` step. I'm not sure if I did something wrong or we have to fix something. Here is the error: 
```  
fill_inflation_restart:  - Reading File : clm_restart.nc
  fill_inflation_restart:  - Reading File : clm_history.nc
 ERROR FROM:
  source : netcdf_utilities_mod.f90
  routine: read_variables: nf90_get_var
  message: TLAI: NetCDF: Start+count exceeds dimension bound
  message: ...
```